tools/source-package-verification.sh

```diff
@@ -5,9 +5,10 @@
 #
 set -e
 
-pip install -r docs/requirements.txt
+pip install -r requirements/requirements-docs.txt
```

Comment:
Only
```suggestion
pip install -r requirements/requirements-tests-install.txt
```
without lines 9, 10, 11 here, then
`pip install -r requirements/requirements-docs.txt`
in the condition below, when calling `make docs`